### PR TITLE
bump annotate-snippets to 0.10

### PR DESCRIPTION
**Pull request by klensy**
# TODO (check if already done)
* [ ] Add tests
* [x] Add CHANGELOG.md entry

### CODE REVIEW

File: src/status_emitter.rs [hunk 1]
```diff
@@ -642,13 +639,9 @@ fn create_error(
             })
             .collect(),
         footer: vec![],
-        opt: FormatOptions {
-            color: colored::control::SHOULD_COLORIZE.should_colorize(),
```
Comment (klensy) [inline src/status_emitter.rs:646]: colors here

**Comment by oli-obk** [comment]
> FIXME: currently there no test fail but should, as previously colors can be switched on\off but not now.

is it detected automatically now? if so, that is fine. If it becomes an issue in tests we can always use a crate for stripping terminal colors

File: src/status_emitter.rs (hunk 1 above)
Comment (klensy) [reply src/status_emitter.rs:646]: From docs (if i understand correctly, https://docs.rs/annotate-snippets/latest/annotate_snippets/struct.Renderer.html#method.plain), `Renderer::plain()` is simple not colored output, while `Renderer::styled()` is some default color scheme.

**Comment by klensy** [comment]
> is it detected automatically now? if so, that is fine. If it becomes an issue in tests we can always use a crate for stripping terminal colors

Nope, with current PR it always not colored, that why fixme added.

**Comment by oli-obk** [comment]
Thanks! With your link and info I was able to get colors back

**Comment by klensy** [comment]
Thanks, forgot about this PR.

**Pull request merged**